packages/datetime2/src/components/date-picker2/_date-picker2.scss

```diff
@@ -0,0 +1,234 @@
+// Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "@blueprintjs/colors/lib/scss/colors";
+@import "@blueprintjs/core/src/components/popover/common";
+@import "@blueprintjs/datetime/src/common";
+
+$cell-size: $pt-grid-size * 3 !default;
```

Comment:
this variable is unused. the dimension is now specified in react-day-picker-overrides as a CSS custom property